spec/features/admin/admin_can_create_an_item_spec.rb

```diff
@@ -26,7 +27,7 @@
     expect(page).to have_content("NewGrossCouch")
   end
 
-  scenario "an admin can create an item without specifying image" do
+  xscenario "an admin can create an item without specifying image" do
```

Comment:
Were these the factory girl things you wanted to talk to @amhursh about?

Reply:
No, I fixed those ones. These are the tests that need to be fixed once we figure out the authorization thing today. Since items now need store id's when an admin creates a new item that new item needs to be assigned the store id that is assigned to that admin.

Reply:
Makes sense. Hopefully we can resolve these later today.